node-runner-cli/commands/dockercommand.py

```diff
@@ -31,26 +31,13 @@ def dockercommand(dockercommand_args=[], parent=docker_parser):
 
 
 @dockercommand([
-    argument("-t", "--trustednode",
-             help="Trusted node on radix network."
-                  "Example format: 'radix://rn1qthu8yn06k75dnwpkysyl8smtwn0v4xy29auzjlcrw7vgduxvnwnst6derj@54.216.99.177'."
-                  "This is required only if you are creating config to run a CORE node and "
-                  "if not provided you will be prompted to enter a value",
-             default="",
-             action="store"),
+    argument("-a", "--autoapprove", help="Set this to true to run without any prompts and in mode CORE or GATEWAY."
```

Comment:
As discussed auto approve on config command can be removed and autoapproval is same as giving prompt feed file